File: official/docs/csharp/v4/addresses/verify-strict-param.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;
using Newtonsoft.Json;
using EasyPost;
using EasyPost.Models.API;

namespace EasyPostExamples
{
    public class Examples
    {
        public static async Task Main()
        {
            var client = new EasyPost.Client("EASYPOST_API_KEY");

            Address address = await client.Address.Create(
                new Dictionary<string, object>
                {
                    { "street1", "417 MONTGOMERY ST" },
                    { "street2", "FLOOR 5" },
                    { "city", "SAN FRANCISCO" },
                    { "state", "CA" },
                    { "zip", "94104" },
                    { "country", "US" },
                    { "company", "EasyPost" },
                    { "phone", "415-123-4567" },
                    { "verify_strict", true }
                }
            );

            Console.WriteLine(JsonConvert.SerializeObject(address, Formatting.Indented));
        }
    }
}

File: official/docs/php/v5/addresses/verify-strict-param.php
<?php

\EasyPost\EasyPost::setApiKey('EASYPOST_API_KEY');

$address = \EasyPost\Address::create([
    'verify_strict'  => true,
    'street1'        => '417 MONTGOMERY ST',
    'street2'        => 'FL 5',
    'city'           => 'San Francisco',
    'state'          => 'CA',
    'zip'            => '94104',
    'country'        => 'US',
    'company'        => 'EasyPost',
    'phone'          => '415-123-4567'
]);

echo $address;
